cylc/flow/scripts/cylc.py

```diff
@@ -248,8 +248,8 @@ def get_version(long=False):
         'cylc print has been removed; use `cylc scan --states=all`',
     'register':
         'cylc register has been removed; use cylc install or cylc play',
-    'reset':
-        'cylc reset has been replaced by cylc set-outputs',
```

Comment:
We should keep this entry and point it at "set"?

Reply:
OK, will do.

Reply:
Done.